MIGRATION.md

```diff
@@ -1,30 +1,36 @@
 # Newman
 
-## Why?
+## Table of contents
 
-Newman v4.0 drops support for Node v4 and deprecated v2 CLI options. Also, the inbuilt HTML reporter has been moved to a standalone reporter.
+1. [Updating to the latest version](#updating-to-the-latest-version)
+2. [Migrating from V3 to V4](#migrating-from-v3-to-v4)
+3. [Migrating from V2 to V3](#migrating-from-v2-to-v3)
 
-## Updating to Newman v4 from older versions
+## Updating to the latest version
 
 If you're updating from an older version of Newman, make sure you read the rest of this document, to understand the changes.
 
-#### 1. Install Newman v4
+#### 1. Update Newman
 ```console
-$ npm install -g newman  # Install newman globally
+$ npm update -g newman
 ```
 
 #### 2. Check Installation
 ```console
 $ newman --version  # Should show the latest version of Newman
```

Comment:
Add `npm show newman version` as a second command here, indicating that both commands should show the same result.